.gitignore

```diff
@@ -1,2 +1,6 @@
 *.swp
 local.toml
+
+# editor files
+.idea/
+vulcan-checks.iml
```

Comment:
Could you and a newline at the end of the file? Helpful when running bash commands/scripts :)

Reply:
fixed :)